### ISSUE TITLE
Black Hole Controller (quest #130) seems uncompletable...

### ISSUE THREAD

**Issue by luke-bravenboer** [imrgr/rise-of-the-pack#12]
... to the average player.

Originally, this was going to be a bug report about how the vanilla ender chest isn't craftable, yet is a component in the recipe of the Black Hole Controller from Industrial Foregoing, required to complete quest #130.

However, after a quick search through the configs, I noticed that you'd intentionally disabled it in the custom crafting scripts, with the comment "_have to go to end city to get them_". I'm guessing you just removed the crafting recipe before you added the Industrial Foregoing questline, rather than intending this to be a secret that players need to figure out themselves? If not, please ignore :)

So this is just a suggestion- maybe mention/hint in the quest blurb that you need to find ender chests naturally (or add the recipe back in)?

**Comment by imrgr** [comment]
Good catch Ligands! I did indeed intend for them to be found in end cities, but you are completely right that the average player would have no way of knowing that. I will add a hint to the quest for the Black Hole Controller in the next update. Thank you for the post!
